src/template_items/templateItem.service.ts

```diff
@@ -22,15 +22,17 @@ export class TemplateItemService {
     const { user_id, limit = 0, offset = 0, filterTag, status = "published" } = getAllTemplates;
     const optionalQuery: { [key: string]: any } = {};
 
-    if (user_id) optionalQuery.user_id = user_id;
-
     if (status !== "all") {
       optionalQuery.status = status;
     } else {
       optionalQuery.status = ["published", "private", "censoring", "illegal"];
     }
 
     if (filterTag) optionalQuery["tags.CourtCategory"] = filterTag;
+    if (user_id) {
+      optionalQuery.user_id = user_id;
+      optionalQuery.status = { $in: ["published", "private", "censoring"] };
+    }
```

Comment:
Why not include _illegal_? Even if the template is rejected by the admin, in the user's MyTemplate page, it should show

Reply:
> Why not include _illegal_? Even if the template is rejected by the admin, in the user's MyTemplate page, it should show

Because this card only fix the 'template cannot display' bug, so I just follow the previous logic.
if BA&PO  think we should add the illegal to the My template page, then I can add as well.